neptune/new/attributes_containers/__init__.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from neptune.new.attributes_containers.attribute_container import AttributeContainer
+from neptune.new.attributes_containers.handler import Handler
```

Comment:
`Handler` is something in the middle between `Attribute` and `AttributeContainer`, I'm not 100% sure about moving it here.